js/fastify/index.js

```diff
@@ -1,2 +1,3 @@
 export * from "./plugins";
+export * from "./auth";

```

Comment:
sort alphabeticly

Reply:
Plugins is a folder, that's why I placed it over there. Sort anyway?